documentation/static/img/blog/2024-08-20-gsoc.png.license

```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: Look Up Look Down Photography
+
+SPDX-License-Identifier: LicenseRef-Unsplash-License
```

Comment:
This should probably point out that this is not available under a open source license but the copyright belongs to google?

Reply:
For this, I made a screenshot of this image from their website, but kinda like being used everywhere